configuration/src/main/java/io/camunda/configuration/Opensearch.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.configuration;
+
+import java.util.Set;
+
+public class Opensearch extends SecondaryStorageDatabase {
+
+  @Override
+  protected String prefix() {
+    return "camunda.data.secondary-storage.opensearch";
```

Comment:
I noticed that some of the prefixes end with dot (like, `Cluster`) and some don't. Shall we have uniform experience?

Reply:
We shall.. I think a prefix doesn't include a dot.. but this might be just an opinion.

Something that pushes me to have prefix without the dot is that sometimes we're using the prefix in the annotations on top of the classes. For example the Camunda class has the @Configuration(prefix = PREFIX). Such prefix doesn't have a dot.

In any case, I don't have a strong feeling for this. If we do this, good. Otherwise, it's fine too

Reply:
I am fine with both format. Actually, my only concern is that it is easy to get yourself confused with dots - happened to me twice already that I forgot to add a dot, so I would personally suggest a utility method `String property(String prefix, String suffix)` or `UnifiedConfigurationHelper.validateLegacyConfiguration(prefix, suffix, field, Type.class, BackwardsCompatibilityMode, Set.of(legacyProp1, legacyProp2))`

Reply:
I think we can remove the dot from everywhere

Reply:
You mean, in prefix?